src/katgpucbf/fgpu/compute.py

```diff
@@ -1,5 +1,5 @@
 ################################################################################
-# Copyright (c) 2020-2023, National Research Foundation (SARAO)
+# Copyright (c) 2020-2024, National Research Foundation (SARAO)
```

Comment:
You've made copyright bumps on a number of files without changing them. Is that left over from the revert?